File: november/08-11/code.js
// start of script.js **

function canPost(message) {

  let postType = "";

  if(message.length <= 40){
    postType = "short post";
  }else if(message.length > 40 && message.length <= 80){
    postType = "long post";
  }else if(message.length > 80){
    postType = "invalid post";
  }

  return postType;
}

// end of script.js **

File: november/08-11/code.py

def can_post(message):
    
    post_type = ""
    if(len(message) <= 40):
        post_type = "short post"
    elif(len(message) > 40 and len(message) <= 80):
        post_type = "long post"
    elif(len(message) > 80):
        post_type = "invalid post"

    return post_type

# end of main.py **

